target/linux/imx/image/cortexa7.mk

```diff
@@ -10,3 +12,21 @@ define Device/Default
   KERNEL_LOADADDR := 0x80008000
   IMAGES :=
 endef
+
+define Device/pico-pi-imx7d
+  DEVICE_VENDOR := TechNexion
+  DEVICE_MODEL := PICO-PI-IMX7D
+  SUPPORTED_DEVICES := technexion,imx7d-pico-pi
+  UBOOT := pico-pi-imx7d
+  DEVICE_DTS := imx7d-pico-pi
+  DEVICE_PACKAGES := kmod-sound-core kmod-sound-soc-imx kmod-sound-soc-imx-sgtl5000 \
+	kmod-can kmod-can-flexcan kmod-can-raw kmod-leds-gpio \
+	kmod-input-touchscreen-edt-ft5x06 kmod-usb-hid kmod-btsdio \
+	kmod-brcmfmac cypress-firmware-4339-sdio cypress-nvram-4339-pico-pi-imx7d
+  FILESYSTEMS := squashfs
+  IMAGES := combined.bin sysupgrade.bin
+  DEVICE_IMG_NAME = $$(DEVICE_IMG_PREFIX)-$$(1).$$(2)
```

Comment:
@Leo-PL any reason why you didn't use generic image filename?

Reply:
Just copy-paste from Apalis, IIRC. This can probably be dropped.